deeppavlov_agent/channels/telegram/bot.py

```diff
@@ -156,31 +157,46 @@ async def handle_dialog_rating(
             callback_query.from_user.id, message_text, reply_markup=reply_markup
         )
 
-    @dp.message_handler(state="*", content_types=["text", "photo"])
+    @dp.message_handler(state="*", content_types=['text', 'photo', 'voice', 'audio', ContentType.VIDEO_NOTE])
     async def handle_message(message: types.Message, state: FSMContext):
         if await state.get_state() == DialogState.active.state:
             message_attrs = {}
-            if message.photo and FILE_SERVER_URL:
+            if message.photo:
                 try:
                     photo = await message.photo[-1].download(BytesIO())
-                    fname = f"{uuid4().hex}.jpg"
+                    fname = f'{uuid4().hex}.jpg'
                     # TODO: make with aiohttp. Maybe remove BytesIO intermediate step. Maybe mobe image logit to agent.
                     # TODO: move file server url definition to the run.py level
-                    resp = requests.post(
-                        FILE_SERVER_URL, files={"file": (fname, photo, "image/jpg")}
-                    )
+                    resp = requests.post(FILE_SERVER_URL, files={'file': (fname, photo, 'image/jpg')})
                     resp.raise_for_status()
-                    download_link = resp.json()["downloadLink"]
-                    download_link = (
-                        urlparse(download_link)
-                        ._replace(scheme=server_url.scheme, netloc=server_url.netloc)
-                        .geturl()
-                    )
-                    message_attrs["image"] = download_link
+                    download_link = resp.json()['downloadLink']
+                    download_link = urlparse(download_link)._replace(scheme=server_url.scheme,
+                                                                     netloc=server_url.netloc).geturl()
+                    message_attrs['image'] = download_link
                 except Exception as e:
                     logger.error(e)
+            sound = message.voice if message.voice else message.audio if message.audio else message.video_note
+            if sound:
+                # FIXME: get_url is not secure — the url contains bot token, that if stolen may be used maliciously
+                sound_message = await sound.get_file()                                    # Multiple audios?
+                # It is guaranteed that the link will be valid for at least 1 hour. When the link expires, a new one can be requested by calling getFile.
+                # Maximum file size to download is 20 MB
```

Comment:
```suggestion
                # It is guaranteed that the link will be valid for at least 1 hour. When the link expires,
                # a new one can be requested by calling getFile. Maximum file size to download is 20 MB.
```
120 characters per line